config.yaml

```diff
@@ -0,0 +1,37 @@
+options:
```

Comment:
this file can now go under charmcraft.yaml in the `config` sections

https://juju.is/docs/sdk/charmcraft-yaml#heading--config

Reply:
done